IntegrationTests/tests_04_performance/test_01_resources/run-nio-alloc-counter-tests.sh

```diff
@@ -18,25 +18,24 @@ here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 tmp_dir="/tmp"
 
-function die() {
```

Comment:
could we leave this in so we get better error messages?

Reply:
Yeah for sure. The reason I removed it is that it didn't seem like it was actually providing better error messages. For example, if you specify an unknown flag, you get the following output:

```
$ ./run-nio-alloc-counter-tests.sh -d
./run-nio-alloc-counter-tests.sh: illegal option -- d
ERROR: unknown option ?
```

The `?` in this case is `$opt`, which is always "?" because it doesn't know what the option is. All of the useful information is contained in the first line, which is the error message from `getopts`.

Removing the only call to `die` (and then the `die` function itself because it's no longer used) gives the following output, which seemed a bit cleaner and less confusing:

```
$ ./run-nio-alloc-counter-tests.sh -d
./run-nio-alloc-counter-tests.sh: illegal option -- d
```

Happy to revert back to the previous code if you prefer though!